scss/style.scss

```
@@ -3,3 +3,261 @@
 @import "_expert.scss";
 @import "_works.scss";
 @import "_footer.scss";
```

Тут можно писать просто:
```suggestion
@import "expert";
@import "works";
@import "footer";
```